microarray/src/org/labkey/microarray/MicroarrayModule.java

```diff
@@ -68,8 +59,6 @@
 
     public static final AssayDataType MAGE_ML_INPUT_TYPE =

```

Comment:
Can any of these other AssayDataTypes go away too?

Reply:
Good catch, I've pruned out a couple more (most notably that MageML) that weren't doing anything. Everything else looks necessary.